package.json

```diff
@@ -43,7 +43,7 @@
     "@octokit/rest": "^16.43.1",
     "@sentry/electron": "^1.2.1",
     "classnames": "^2.2.6",
-    "electron-default-menu": "^1.0.1",
+    "electron-default-menu": "1.0.2",
```

Comment:
```suggestion
    "electron-default-menu": "^1.0.2",
```

Reply:
I re-read the caret did and indeed I shouldn't have pinned it.